assets/aml-benchmark/components/batch_benchmark_inference/spec.yaml

```diff
@@ -164,52 +193,68 @@ jobs:
         type: mltable
       ground_truth_metadata:
         type: uri_folder
-  # Inference
-  endpoint_batch_score:
-    type: parallel
-    component: azureml:batch_benchmark_score:0.0.6
+  # Config generation
+  config_generation:
+    type: command
+    component: azureml:batch_benchmark_config_generator@latest
     inputs:
-      model_type: ${{parent.inputs.model_type}}
-      online_endpoint_url: ${{parent.inputs.endpoint_url}}
+      scoring_url: ${{parent.inputs.endpoint_url}}
+      connection_name: ${{parent.inputs.connections_name}}
       deployment_name: ${{parent.inputs.deployment_name}}
-      connections_name: ${{parent.inputs.connections_name}}
+      authentication_type: ${{parent.inputs.authentication_type}}
       debug_mode: ${{parent.inputs.debug_mode}}
       additional_headers: ${{parent.inputs.additional_headers}}
       ensure_ascii: ${{parent.inputs.ensure_ascii}}
       max_retry_time_interval: ${{parent.inputs.max_retry_time_interval}}
       initial_worker_count: ${{parent.inputs.initial_worker_count}}
       max_worker_count: ${{parent.inputs.max_worker_count}}
+      configuration_file: ${{parent.inputs.endpoint_config_file}}
+      model_type: ${{parent.inputs.model_type}}
+      app_insights_connection_string: ${{parent.inputs.app_insights_connection_string}}
+    outputs:
+      batch_score_config:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.json
+  # Inference
+  endpoint_batch_score:
+    type: parallel
+    component: azureml:batch_score_llm@latest
+    inputs:
+      async_mode: False
       data_input_table: ${{parent.jobs.batch_inference_preparer.outputs.formatted_data}}
-      mini_batch_size: ${{parent.inputs.mini_batch_size}}
-      deployment_metadata: ${{parent.inputs.endpoint_config_file}}
+      configuration_file: ${{parent.jobs.config_generation.outputs.batch_score_config}}
     outputs:
-      job_out_path:
+      job_output_path:
         type: uri_file
-      mini_batch_results_out_directory:
-        type: uri_folder
-      metrics_out_directory:
+      mini_batch_results_output_directory:
         type: uri_folder
     resources:
       instance_count: ${{parent.inputs.instance_count}}
     max_concurrency_per_instance:  ${{parent.inputs.max_concurrency_per_instance}}
-    mini_batch_size: "100KB"
+    mini_batch_size: 100KB #${{parent.inputs.mini_batch_size}}
     retry_settings:
       timeout: 6000
       max_retries: 10
+    environment_variables:
+      BATCH_SCORE_INITIAL_REQUEST_TIMEOUT: '4'
+      BATCH_SCORE_DELAY_AFTER_SUCCESSFUL_REQUEST: 'False'
+      BATCH_SCORE_MAX_REQUEST_TIMEOUT: '300'
+      
   # Reformat
   batch_output_formatter: 
     type: command
-    component: azureml:batch_output_formatter:0.0.7
+    component: azureml:batch_score_llm@latest
```

Comment:
Looks like component name was replaced by mistake.

Reply:
I have updated it. Thanks for catching this!